persistence/src/main/scala/com/mesosphere/usi/state/plugin/PathId.scala

```diff
@@ -0,0 +1,23 @@
+package com.mesosphere.usi.state.plugin
+//package plugin
+
+import scala.collection.immutable.Seq
+
+/**
+  * A path based identifier.
+  */
+trait PathId {
```

Comment:
I don't think we will follow this pattern. Eg the Jenkins plugin will probably use some kind of storage but not a `PathId`.

Reply:
Removed